terraform-provider-constellation/docs/resources/cluster.md

```diff
@@ -79,7 +78,7 @@ resource "constellation_cluster" "azure_example" {
 
 - `api_server_cert_sans` (List of String) List of Subject Alternative Names (SANs) for the API server certificate. Usually, this will be the out-of-cluster endpoint and the in-cluster endpoint, if existing.
 - `azure` (Attributes) Azure-specific configuration. (see [below for nested schema](#nestedatt--azure))
-- `constellation_microservice_version` (String) The version of Constellation's microservices used within the cluster. When not set, the provider default version is used.
+- `constellation_microservice_version` (String) The version of Constellation's microservices used within the cluster. When not set, version v0.0.0 is used.
```

Comment:
will the docs update during the release?

Reply:
Good point, let's leave this out for now to be safe here.